packages/lodestar/src/chain/chain.ts

```diff
@@ -215,6 +217,13 @@ export class BeaconChain extends (EventEmitter as { new(): ChainEventEmitter })
     genesisBlock.stateRoot = stateRoot;
     const blockRoot = signingRoot(this.config.types.BeaconBlock, genesisBlock);
     this.latestState = genesisState;
+    // Determine whether a genesis state already in
+    // the database matches what we were provided
+    const storedGenesisBlock = await this.db.block.getBlockBySlot(GENESIS_SLOT);
```

Comment:
where is GENESIS_SLOT defined?

shouldn't this be this.config.params.GENESIS_SLOT

Reply:
It’s a [constant that was imported](https://github.com/aunyks/lodestar/blob/a4abbd9f9acad4b8f6e46f92d1ead186800c0b36/packages/lodestar/src/chain/chain.ts#L11).

Reply:
Could you try to remove that constant from that file and replace occurencies with this.config.params.GENESIS_SLOT. Not sure in what case genesis slot could be different from 0 but nevertheless

Reply:
I think that can be done separately, as needed. It may touch a bunch of other files and have some consequences we haven't fully thought through.